### PR TITLE
[13_0 backport]: 2023 FastSim Workflow

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
Backport of https://github.com/cms-sw/cmssw/pull/40954

#### PR validation:
Test with workflow 14034.0.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:
This is a backport PR.

### CODE REVIEW

**Comment by srimanob** [comment]
test parameters:
- workflows = 14034.0
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 1]
```diff
@@ -2314,6 +2316,14 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'BeamSpot': 'Realistic25ns13p6TeVEarly2022Collision',
         'ScenToRun' : ['GenSim','Digi','RecoNano','HARVESTNano','ALCA'],
     },
+    '2023FS' : {
+        'Geom' : 'DB:Extended',
+        'GT' : 'auto:phase1_2023_realistic',
+        'HLTmenu': '@relval2022',
```
Comment (missirol) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:2322]: This should eventually change to `HLT:@relval2023` in light of #41017 (and https://github.com/cms-sw/cmssw/pull/41007). If this PR is integrated before those, #41017 will need to be rebased to change this to `@relval2023`.

**Review by missirol — COMMENTED** [review]
This PR is in a bit of a clash with https://github.com/cms-sw/cmssw/pull/41017.

Once one of the two is merged, the other one will need to be rebased.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py (hunk 1 above)
Comment (srimanob) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:2322]: Thanks @missirol 
I will let #41017 and #41007 merge first. This PR can be rebases, it is very small.

**Comment by srimanob** [comment]
hold

Waiting #41017 and #41007 to be merged first.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py (hunk 1 above)
Comment (missirol) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:2322]: Thanks, @srimanob ! Could you please also have a look at #41016 (needs upgrade signature) ?

**Comment by srimanob** [comment]
unhold

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

The failure in comparison is on [11634.7_TTbar_14TeV+2021_trackingMkFit/](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-32545a/31399/runTheMatrix-results/11634.7_TTbar_14TeV+2021_trackingMkFit/) which should not relate to this PR.

**Comment by kskovpen** [comment]
+pdmv

**Comment by perrotta** [comment]
backport of https://github.com/cms-sw/cmssw/pull/40954

**Comment by perrotta** [comment]
+1

**Pull request merged**